core/store/migrate/migrations/0188_create_automation_upkeep_state.sql

```diff
@@ -0,0 +1,21 @@
+-- +goose Up
+
+CREATE TABLE evm_upkeep_states (
+    work_id TEXT PRIMARY KEY,
+    evm_chain_id NUMERIC NOT NULL,
+    upkeep_id BYTEA NOT NULL,
+    completion_state SMALLINT NOT NULL,
+    ineligibility_reason NUMERIC NOT NULL,
+    block_number NUMERIC,
```

Comment:
I would suggest using `int64` everywhere for block_number, as geth does.

Reply:
So far it was chosen to use `uint64` by the automation team since block numbers are not negative even though geth uses `int64` internally.